posthog/models/short_link.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.utils import timezone
+from posthog.models.team import Team
+
+
+class ShortLink(models.Model):
+    """
+    Links that redirect to a specified destination URL.
+    These are used for sharing URLs across the application.
+    """
+
+    id = models.CharField(max_length=36, primary_key=True)
+    destination = models.URLField(max_length=2048)
+    origin_domain = models.CharField(max_length=255)
+    origin_key = models.CharField(max_length=255)
+    team = models.ForeignKey(Team, on_delete=models.CASCADE)
+    created_at = models.DateTimeField(default=timezone.now)
+    updated_at = models.DateTimeField(auto_now=True)
+    description = models.TextField(null=True, blank=True)
+    tags = models.TextField(null=True, blank=True)
+    comments = models.TextField(null=True, blank=True)
+
+    class Meta:
+        constraints = [models.UniqueConstraint(fields=["id", "team_id"], name="unique_short_link_id_per_team")]
```

Comment:
There's no constraint on short link. We also don't need constraint on id/team_id, this is useless (there's already constraint on id by default because it's pk)